index.html

```diff
@@ -2723,66 +2709,63 @@ <h2>Protocol Bindings</h2>
+                enable the behavior implementation to instantiate <a>Consumed Things</a>
+                that internally interface with the system instead of the protocol stack.
+                This can be done by listing such system Things in the discovery results.
+            </p>
+            <p>
+                A device may also be physically external to a <a>Servient</a>,
+                but connected via a proprietary protocol or a protocol
+                not eligible as <a>WoT Interface</a>. In
+                this case, the <a>WoT Runtime</a> may
                 access legacy devices with such protocols (e.g., Echonet
                 Lite, BACNET, X10, I2C, SPI, etc.) through proprietary APIs,
-                but may again choose to expose them to applications via
+                but may again choose to expose them to the behavior implementation via
                 a <a>Thing</a> abstraction. A <a>Servient</a> can then act
                 as gateway to the legacy devices. This should only be
                 done if the legacy device cannot be described directly
                 using a <a>WoT Thing Description</a>.
             </p>
+            <p>
+                The behavior implementation may also access local
+                hardware or system services (e.g., storage) through a
+                proprietary API or other means. This is, however, out of
+                scope of the W3C WoT standardization, as it hinders portability.
+            </p>
         </section>
         <section id="alt-servient-impl">
-            <h3>Alternative Servient Implementations</h3>
-            <p> The <a>WoT Scripting API</a> building block is optional.
-                Alternative <a>Servient</a> implementations are possible
+            <h3>Alternative Servient and WoT Implementations</h3>
+            <p>
+                The <a>WoT Scripting API</a> building block is optional.
+                Alternative <a>Servient</a> implementations are possible,
                 where applications are implemented using alternative
-                APIs internally. There are two possibilities for such
-                implementations: those that use the <a>WoT Runtime</a> but use
-                an alternative API and programming language for defining
-                an application; and those that do not use a <a>WoT Runtime</a>
-                but are implemented using a separate software stack. The
-                second case can still be incorporated into a WoT system
-                if they use a WoT-supported protocol and a Thing
-                Description can be provided for such devices or services.
+                northbound APIs for the <a>WoT Runtime</a> or provide the
+                behavior implementation in a native programming language.
+            </p>
+            <p>
+                Furthermore, devices or services unaware of W3C WoT can still be consumed,
```

Comment:
unaware of WoT = not implementing any building block? Does this have any implications to a consumer?

Reply:
No, as long as the second part of the sentence is fulfilled.